ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -7,15 +7,18 @@ chain:
     - name: HYPERSHIFT_NODE_COUNT
       default: "3"
       documentation: "The number nodes to automatically create and join to the cluster."
+    - name: HYPERSHIFT_CLI
+      default: "hypershfit"
+      documentation: "The chosen CLI, hcp or hypershift"
     commands: |-
       set -exuo pipefail
 
       HCP_CLI="/usr/bin/hcp"
-      if [[ ! -f $HCP_CLI ]]; then
+      if [[ ! -f $HCP_CLI && "${HYPERSHIFT_CLI}" == "hcp" ]]; then
```

Comment:
this doesn't seem accurate. If the `HYPERSHIFT_CLI` env var is set to `hcp`, then why would we want to set it to use the `hypershift` cli instead?